aphrodite/modeling/layers/sampler.py

```diff
@@ -241,24 +254,28 @@ def _get_temperatures(input_metadata: InputMetadata) -> List[float]:
     return temperatures
 
 
-def _get_top_p_top_k(
+def _get_top_ap_top_k(
     input_metadata: InputMetadata,
     vocab_size: int,
-) -> Tuple[List[float], List[int]]:
+) -> Tuple[List[float], List[int], List[float]]:
     top_ps: List[float] = []
     top_ks: List[int] = []
+    top_as: List[float] = []
     for seq_group in input_metadata.seq_groups:
         seq_ids, sampling_params = seq_group
-        top_p = sampling_params.top_p
         # k should not be greater than the vocab size.
         top_k = min(sampling_params.top_k, vocab_size)
         # k=-1 means no truncation.
         top_k = vocab_size if top_k == -1 else top_k
-        top_ps += [top_p] * len(seq_ids)
+
+        top_ps += [sampling_params.top_p] * len(seq_ids)
         top_ks += [top_k] * len(seq_ids)
-    return top_ps, top_ks
+        top_as += [sampling_params.top_a] * len(seq_ids)
+
+    return top_ps, top_ks, top_as
 
 
+def _apply_top_ap_top_k(
```

Comment:
looks like something got messed up here

Reply:
No typo, it represents that the Top-A and Top-P samplers were merged into a single pass.
Possibly confusing, but easy to fix.

Reply:
lgtm